### PR TITLE
docs: add article

### PR DESCRIPTION
**Pull request by Atinux**

### CODE REVIEW

File: docs/content/5.blog/announcing-v2.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- **MDC:** Markdown Components
+- Multi sources (experimental)
+- Locale support (i18n)
+- Navigation generation
+- Live preview edition (coming soon)
+
+The repository is open source under the MIT license and available on GitHub: [nuxt/content](https://github.com/nuxt/content)
+## What is Nuxt Content?
+
+Nuxt Content is a [Nuxt module](https://modules.nuxtjs.org) allowing you to have a `content/` directory where you can place your Markdown, YAML, CSV and JSON files.
```
Comment (pi0) [inline docs/content/5.blog/announcing-v2.md:37]:
```suggestion
Nuxt Content is a [Nuxt module](https://v3.nuxtjs.org/guide/features/modules) allowing you to have a `content/` directory where you can place your Markdown, YAML, CSV and JSON files.
```

File: docs/content/5.blog/announcing-v2.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+2 years after the [release of Content v1](https://github.com/nuxt/content/releases/tag/v1.0.0), we are proud to announce the second version of Nuxt Content with the support of [Nuxt 3](https://v3.nuxtjs.org).
+
+It has been rewritten in TypScript and comes with new features:
+
+- **MDC:** Markdown Components
+- Multi sources (experimental)
+- Locale support (i18n)
+- Navigation generation
+- Live preview edition (coming soon)
```
Comment (pi0) [inline docs/content/5.blog/announcing-v2.md:32]: Runtime server support is kinda missing. It is one of the major changes between v1 and v2

File: docs/content/5.blog/announcing-v2.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- Props had to live in document front-matter = no scope
+- No inline components were possible combined with Markdown
+
+In Content 2, we created a new Markdown syntax for leveraging the power of Vue components and fixing all the issues above, introducing the MDC syntax.
+
+First, MDC is Markdown, so nothing changes and you can keep using the .md extension.
+
+Next, the HTML tags are replaced with a colon-like naming convention:
+
+<my-component /> becomes :my-component (or now possible :MyComponent)
```
Comment (pi0) [inline docs/content/5.blog/announcing-v2.md:71]:
```suggestion
`<my-component />` becomes `:my-component` (or now possible `:MyComponent`)
```

File: docs/content/5.blog/announcing-v2.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+If you need to use slots, you can the the :: syntax:
+
+<my-button>hello</my-button> becomes :my-button[hello]
```
Comment (pi0) [inline docs/content/5.blog/announcing-v2.md:75]:
```suggestion
`<my-button>hello</my-button>` becomes `:my-button[hello]`
```

File: docs/content/5.blog/announcing-v2.md (hunk 2 above)
Comment (Atinux) [reply docs/content/5.blog/announcing-v2.md:32]: It was also working in content v1 with `nuxt start` actually

Comment (pi0) [reply docs/content/5.blog/announcing-v2.md:32]: Yes but implementation is runtime...

**Pull request merged**